pipe-cli/mount/pipe-fuse.py

```diff
@@ -342,6 +346,7 @@ def parse_mount_options(options_string):
     parser.add_argument("-th", "--threads", action='store_true', help="Enables multithreading.")
     parser.add_argument("-d", "--monitoring-delay", type=int, required=False, default=600,
                         help="Delay between path lock monitoring cycles.")
+    parser.add_argument("--show-archived", action='store_true', help="Show archived files.")
     args = parser.parse_args()
```

Comment:
I would recommend adding additional `--storage-class-exclude=GLACIER,DEEP_ARCHIVE` argument which will allow skipping certain storages classes. It can be used along with `--show-archived=False` to completely ignore glacier archived/unarchived files.